src/Selenium2Library/locators/windowmanager.py

```diff
@@ -139,8 +140,17 @@ def _select_matching(self, browser, matcher, error):
         raise ValueError(error)
 
     def _get_current_window_info(self, browser):
-        id_, name, title, url = browser.execute_script(
-            "return [ window.id, window.name, document.title, document.URL ];")
+        try:
+            id_, name = browser.execute_script("return [ window.id, window.name ];")
+        except WebDriverException:
```

Comment:
It's alright. At least it indicates that some sort of selenium error did happen and it is, in this case, OK to suppress the error.